.gitignore

```diff
@@ -69,3 +69,6 @@ webapp/snapcraft_api/
 .venv/
 
 .coverage
+scripts/regenerate-cve-migrations-postgres.sh
```

Comment:
I think this can go.

Reply:
yep

Reply:
Moving this [to `.git/info/exclude`](https://help.github.com/en/github/using-git/ignoring-files#excluding-local-files-without-creating-a-gitignore-file).